src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -1212,6 +1212,10 @@ public override void StatusEventHandler(object sender, BuildStatusEventArgs e)
                     propertyOutputMap[evaluationKey] = value;
                 }
             }
+            else if (e is BuildCanceledEventArgs buildCanceled)
+            {
+                Console.WriteLine(e.Message ?? string.Empty);
```

Comment:
Is `string.Empty` the right default, or should we ensure that `BuildCanceledEventArgs.Message` is never empty?

Reply:
This was more for making nullability check happy. Now I ensure there will be a message in event ctor.